src/component/card/card.tsx

```diff
@@ -3,8 +3,9 @@ import './card.scss'
 
 export type CardProps = {
   children: React.ReactElement
+  className?: string
 }
 
-export const Card: FC<CardProps> = ({ children }) => {
-  return <div className="okp4-dataverse-portal-card-main">{children}</div>
+export const Card: FC<CardProps> = ({ children, className }) => {
+  return <div className={`okp4-dataverse-portal-card-main ${className}`}>{children}</div>
```

Comment:
Works, but not correct. If className is empty, the value of the className property will be `okp4-dataverse-portal-card-main ` with a space at the end.

Reply:
Nice catch ⚾ 🧤 !